lib/code_ocean/config.rb

```diff
@@ -2,12 +2,18 @@
 
 module CodeOcean
   class Config
-    def initialize(filename)
+    attr_reader :filename, :path, :read, :options
+
+    def initialize(filename, options = {})
       @filename = filename
+      @options = options
```

Comment:
If the options and filename are not used (yet), I tend to not expose them.

Reply:
👍, I've changed this and only export the `path` and content (via `read`) now.